### PR TITLE
ldc 1.4.0

### PR DESCRIPTION
**Pull request by John-Colvin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ilovezfs** [comment]
@John-Colvin it seems the test fails.

**Comment by ilovezfs** [comment]
The test is still failing with the updated bootstrapper.

**Comment by sambrightman** [comment]
These two releases add support for LLVM 5.0, so I believe the dependency on `llvm@4` can be changed to the default `llvm`. I have sporadic build failures though, in addition to the failing test.

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by John-Colvin** [comment]
@BrewTestBot test this please

**Comment by John-Colvin** [comment]
The tests all passed except for one network problem on one run, which was `wiki.dlang.org` going down temporarily.

**Comment by John-Colvin** [comment]
@BrewTestBot test this please

File: Formula/ldc.rb [hunk 1]
```diff
@@ -42,17 +40,19 @@ def install
...
         system "make"
       end
     end
     mkdir "build" do
       args = std_cmake_args + %W[
-        -DLLVM_ROOT_DIR=#{Formula["llvm@4"].opt_prefix}
+        -DLLVM_ROOT_DIR=#{Formula["llvm"].opt_prefix}
         -DINCLUDE_INSTALL_DIR=#{include}/dlang/ldc
         -DD_COMPILER=#{buildpath}/ldc-lts/build/bin/ldmd2
+        -DLDC_WITH_LLD=OFF
```
Comment (ilovezfs) [inline Formula/ldc.rb]: what do these two new args do?

Comment (John-Colvin) [reply Formula/ldc.rb]: see added comments below

**Comment by John-Colvin** [comment]
this is good to go I think

**Comment by John-Colvin** [comment]
woops, had the comments the wrong way around, good now

**Comment by John-Colvin** [comment]
@BrewTestBot test this please

**Comment by John-Colvin** [comment]
Ping?

**Comment by ilovezfs** [comment]
Shipped! Thanks @John-Colvin and sorry for the wait

**Pull request closed without merging**